docs/usage/pivoting.md

```diff
@@ -1,73 +1,80 @@
 
 ![Top menu investigation](assets/top-menu-investigation.png)
 
-Investigations are organized by workspace. When you create a new empty workspace, it will only be visible by you and enables you to work on your investigation before sharing it.
+!!! note "Access restriction"
 
-In your workspace, you can add entities that you want to investigate, visualize the data linked to these entities, add relationships, and export your investigation graph in pdf, image or as new stix report.
+    When an investigation is created, it is initially visible only to the creator, allowing them to work on the investigation before deciding to share it. The sharing mechanism is akin to that of dashboards. For further details, refer to the [Access control section](dashboards.md#access-control-section) in the dashboard documentation page.
 
-![Investigation workspace](assets/investigation-workspace.png)
 
-You can see next to them a bullet with a number inside. It is a visual indication showing you how many entities are linked to this one and not displayed in the graph yet.
-Note that this number is an approximation of the number of entities. That's why there is a `~` next to the number.
+## Perform investigation
 
-No bullet displayed means there is nothing to expand from this node.
 
-## Add and expand an entity
+### Manipulate entity
 
-You can add any existing entity of the platform to your investigation.
+We can add any existing entity of the platform to your investigation.
 
 ![Investigation bottom right menu](assets/investigation-bottom-right-menu.png)
 
-Once added, you can select the entity, and see its details in the panel that appears on the right of the screen.
+After adding an entity, we can choose the entity and view its details in the panel that appears on the right of the screen.
 
+On each node, we'll notice a bullet with a number inside, serving as a visual indication of how many entities are linked to it but not currently displayed in the graph. Keep in mind that this number is an approximation, which is why there's a "+" next to it. If there's no bullet displayed, it means there's nothing to expand from this node.
 
-In the same menu as above, right next to "Add en entity", you can expand the selected entity. Clicking on the menu icon open a new window where you can choose which type of entities and relationships you want to expand.
+![Investigation workspace](assets/investigation-workspace.png)
 
-For each type of entity or relationship, the number of elements that will be added into the investigation graph is displayed in parentheses. This time there is no `~` symbol as the number is exact.
+To incorporate these linked entities into the graph, we just have to expand the nodes. Utilize the button with a 4-arrows logo in the menu mentioned menu, or double-click on the entity directly. This action opens a new window where we can choose the types of entities and relationships we wish to expand.
```

Comment:
'menu mentioned menu' is that a duplicate?

Reply:
Indeed, good catch, thank you